corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -489,7 +489,7 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search and Claim Options" %
             </div>
           </div>
           {% endif %}
-          {% if request|toggle_enabled:'SPLIT_SCREEN_CASE_SEARCH' and not app.split_screen_dynamic_search%}
+          {% if split_screen_case_search and not app.split_screen_dynamic_search %}
```

Comment:
nit: having `_enabled` in that name would make it easier to understand what this variable is.

Reply:
I named it "split_screen_case_search" to match "app.split_screen_dynamic_search". It feels out of scope for this FF to rename "app.split_screen_dynamic_search" to  "app.split_screen_dynamic_search_enabled". Do you think I should keep them matching, or rename the one and not the other?

Reply:
I was thinking of `split_screen_case_search_enabled` which was added in this PR.